src/model/image-environment-factory.ts

```
@@ -82,17 +83,12 @@
-    if (parameters.providerStrategy === 'local-docker') {
-      for (const element of additionalVariables) {
-        if (!environmentVariables.some((x) => element?.name === x?.name)) {
-          environmentVariables.push(element);
-        }
-      }
-      for (const variable of environmentVariables) {
-        if (!environmentVariables.some((x) => variable?.name === x?.name)) {
-          environmentVariables = environmentVariables.filter((x) => x !== variable);
-        }
-      }
+    /* eslint-disable-next-line lines-around-comment */
```

ESLint

[error] 'lines-around-comment' rule is disabled but never reported.

(eslint-comments/no-unused-disable)